docs/source/reference/yaml-spec.rst

```diff
@@ -153,6 +156,18 @@ Available fields:
     #
     # Values set here can be overridden by a CLI flag:
     # `sky launch/exec --env ENV=val` (if ENV is present).
+    #
+    # If you want to use a docker image in a private registry, you need to specify your
+    # username, password and repository URI in task envs. For example:
+    #   envs:
+    #     DOCKER_USERNAME: <username>
+    #     DOCKER_PASSWORD: <password>
+    #     DOCKER_REPO_URI: <registry URI>
```

Comment:
QQ: where does this `REPO_URI` term come from? Should we use `DOCKER_SERVER` instead, as docker doc? https://docs.docker.com/engine/reference/commandline/login/

Reply:
It is from the issue 🤧 changed to server now